hadoop-hdds/rocksdb-checkpoint-differ/src/test/java/org/apache/ozone/rocksdiff/TestRocksDBCheckpointDiffer.java

```diff
@@ -1794,7 +1794,7 @@ public void testShouldSkipNode(Map<String, String> columnFamilyToPrefixMap,
         .getCompactionNodeMap().values().stream()
         .sorted(Comparator.comparing(CompactionNode::getFileName))
         .map(node ->
-            rocksDBCheckpointDiffer.shouldSkipNode(node,
+            RocksDiffUtils.shouldSkipNode(node,
```

Comment:
Tests (`testShouldSkipNode` and `testShouldSkipNodeEdgeCase`) should be moved to `TestRocksDiffUtils`. If not part of this PR, create a follow-up task.